src/components/Content/subpages/Surveys/Surveys.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import MyTable from "../../../reusable/MyTable";
+import moment from "moment";
+
+const Surveys = () => {
+  const data = [
+    {
+      key: "1",
+      name: "John Brown",
+      age: 32,
+      address: "New York No. 1 Lake Park",
+      date: moment("2019-12-25").format("YYYY-MM-DD"),
+      showContent: false
+    },
+    {
+      key: "2",
+      name: "Aim Green",
+      age: 42,
+      address: "London No. 1 Lake Park",
+      date: moment("2000-12-22").format("YYYY-MM-DD"),
+      showContent: false
+    },
+    {
+      key: "3",
+      name: "Coe Black",
+      age: 32,
+      address: "Sidney No. 1 Lake Park",
+      date: moment("2012-12-25").format("YYYY-MM-DD"),
+      showContent: false
+    },
+    {
+      key: "4",
+      name: "Xim Red",
+      age: 32,
+      address: "London No. 2 Lake Park",
+      date: moment("2018-10-25").format("YYYY-MM-DD"),
+      showContent: false
+    }
+  ];
+
+  const columns = [
+    {
+      key: "key",
+      column: "Key"
+    },
+    {
+      key: "name",
+      column: "Name"
+    },
+    {
+      key: "age",
+      column: "Age"
+    },
+    {
+      key: "address",
+      column: "Address"
+    },
+    {
+      key: "date",
+      column: "Date"
+    }
+  ];
+  return (
+    <div>
+      Surveys
+      <MyTable columns={columns} data={data} />
```

Comment:
Tip: We prefer to use JSX as much as possible, so the perfect API for the table in my mind would look like:
```jsx
<Table
  rows={[
    { id: '1', name: 'Name1', state: 'state1', audience: 'audience1', date: new Date },
    { id: '2', name: 'Name2', state: 'state2', audience: 'audience2', date: new Date },
    { id: '3', name: 'Name3', state: 'state3', audience: 'audience3', date: new Date }
  ]}
  rowIdAccessor={row => row.id}
  rowActionsComp={() => (
    <React.Fragment>
      <DropdownMenu.Item
        label="Button 1"
        onClick={() => {}}
      />
      <DropdownMenu.Item
        label="Button disabled"
        isDisabled
        onClick={() => {}}
      />
      <DropdownMenu.Item
        label="Button 3"
        onClick={() => {}}
      />
    </React.Fragment>
  )}
>
  <Table.Head>
    <Table.TH flex={3} sortKey="name">Name</Table.TH>
    <Table.TH flex={2} sortKey="state">State</Table.TH>
    <Table.TH flex={2} sortKey="audience">Audience</Table.TH>
    <Table.TH flex={2} sortKey="date">Date</Table.TH>
    <Table.TH flex={2} align="center" sortKey="importedByImageUrl">Imported by</Table.TH>
  </Table.Head>
  <Table.Body>
    <Table.TR>
      {({ name, state, audience, date, importedByImageUrl }) =>
        <React.Fragment>
          <Table.TD flex={3} isPrimary>{name}</Table.TD>
          <Table.TD flex={2}>{state}</Table.TD>
          <Table.TD flex={2}>{audience}</Table.TD>
          <Table.TD flex={2}>{date.toLocaleDateString()}</Table.TD>
          <Table.TD align="center" flex={2}>
            <Avatar.S />
          </Table.TD>
        </React.Fragment>}
    </Table.TR>
  </Table.Body>
</Table>
```
If you need for example add image to one column or add some specific style to one cell it's much easier to achive it with API like this.

Reply:
I have seen on that approach and agree with you about specific style, but if I consider reusability of your Table component, maybe it could be better - it means, there are hardcoded columns  `{({ name, state, audience, date, importedByImageUrl })` and rows data.
If it will be request to change it like this, so I have no problem to do it :)

Reply:
There aren't any hardcoded columns, everything is flexible. What do you get in `{({ name, state, audience, date, importedByImageUrl })` depends on what do you pass into the `rows` prop at `Table` component.
```js
// if you pass
rows={[
    { id: '1', name: 'Name1', state: 'state1', audience: 'audience1', date: new Date },
    { id: '2', name: 'Name2', state: 'state2', audience: 'audience2', date: new Date },
    { id: '3', name: 'Name3', state: 'state3', audience: 'audience3', date: new Date }
  ]}
// then you get
{({ id, name, state, audience, date })

// if you pass
rows={[
    { id: '1', email: 'Name1' },
    { id: '2', email: 'Name2' },
    { id: '3', email: 'Name3' }
  ]}
// then you get
{({ id, email })
```